Compfiles/Imo1978P5.lean

```diff
@@ -0,0 +1,643 @@
+/-
+Copyright (c) 2025 Roozbeh Yousefzadeh. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Roozbeh Yousefzadeh
+-/
+
+import Mathlib
+import Mathlib.Tactic
+
+import ProblemExtraction
+
+problem_file { tags := [.Algebra] }
```

Comment:
If you like, you can put a link to your original repo here.
Like this:
https://github.com/dwrensha/compfiles/blob/487db7cf02a5dc779f442b49eb50d01ba3c523a2/Compfiles/Imo2024P6.lean#L13-L14